src/scripts.js

```diff
@@ -2,19 +2,25 @@
 let userInfo = document.querySelector(".user-info");
 let displayMessage = document.querySelector(".display-message");
 let stepGoalComparison = document.querySelector(".step-goal-comparison");
+let hydrationWeekDisplay = document.querySelector(".hydration-week");
+let hydrationTodayDisplay = document.querySelector(".hydration-today");
 
 //Event Listeners
 window.addEventListener("load", initializeUserInfo);
 
 //Global Variables
-let activeUser = new User(sampleData[1]);
-let userRepo = new UserRepository(sampleData);
+let activeUser = new User(sampleUserData[1]);
```

Comment:
Is this the user that will load by default every time someone goes to the website? You may have plans to change this, but what sort of logic could you use to have a random user populate on the website each time it is loaded?

Reply:
Correct, for the time being we were just using a set person to make sure we were accessing the data correctly. We can use a function to randomize the user and set it to be a currentUser or something. Will probably be adding this in after we finish building out the Sleep and Activity classes